t/26-ssh_screen.t

```diff
@@ -8,9 +8,26 @@ use FindBin '$Bin';
 use lib "$Bin/../external/os-autoinst-common/lib";
 use OpenQA::Test::TimeLimit '5';
 use consoles::ssh_screen;
+use Test::Exception;
+use Test::MockObject;
+use Test::MockModule;
 
 my $screen = consoles::ssh_screen->new(ssh_connection => 'My_Con', ssh_channel => 'My_Chan');
 is($screen->{fd_read}, 'My_Chan', 'SSH channel is used for reading');
 is($screen->{fd_write}, 'My_Chan', 'SSH channel is used for writing');
 
+subtest 'Correct message when type_string timeouts' => sub {
+    use consoles::serial_screen;
+    use bmwqemu;
```

Comment:
I'd prefer to move them to the top. They're already loaded at compile time anyways.

Reply:
Done